src/ucs/config/global_opts.h

```diff
@@ -27,96 +27,96 @@ BEGIN_C_DECLS
  */
 typedef struct {
 
-    /* Log level above which log messages will be printed */
-    ucs_log_level_t          log_level;
+    /* Log level above which log messages will be printed for default component*/
```

Comment:
add a whitespace at the end of the comment:
`/* ...  for default component */`

Reply:
done